assets/data/cards_pt-br.json

```
@@ -986,6 +1005,15 @@
       ],
       "description": "Um StatelessWidget, no Flutter, é um componente que não possui estado interno mutável, ideal para partes da interface do usuário que não mudam com o tempo e não dependem de estado."
     },
+    {
+      "title": "Svelte",
+      "tags": [
+        "Conceito",
+        "Front-end",
+        "Framework"
+      ],
+      "description": "Svelte é um framework JavaScript que compila o código em tempo de build, eliminando a necessidade de uma camada extra no navegador, o que o torna mais eficiente e leve em comparação a outros frameworks."
```

Simplifiquei a explicação sobre a "compilação em tempo de build" e o impacto no desempenho. O que acha?

```suggestion
      "description": "Svelte é uma ferramenta JavaScript que transforma o código durante a construção do site ou aplicativo, sem precisar de uma camada extra para funcionar no navegador. Isso o torna mais rápido e leve em comparação com outras ferramentas similares."
```